src/index.js

```diff
@@ -92,10 +91,6 @@ webext.startup().then(({ browser }) => {
       }
     }
   });
-  setTimeout(
-    () => {
-      new Notification({ surveyUrl: storage.surveyUrl });
-    },
-    5000
-  );
+
+  new NotificationWatcher();
```

Comment:
I specifically don't want this to be GCed, but in this context there's nothing permanent to which to assign it (i.e. `window`).